packages/uikit-react-native/src/components/ChannelMessageList/index.tsx

```diff
@@ -325,31 +293,34 @@ const useGetMessagePressActions = <T extends SendbirdGroupChannel | SendbirdOpen
             });
         },
       });
-
-      if (!channel.isEphemeral) {
-        if (isMyMessage(msg, currentUserId) && msg.sendingStatus === 'succeeded') {
-          sheetItems.push({
-            // TODO: disabled if message has a parentMessageId
-            // disabled: Boolean(msg.parentMessageId),
-            icon: 'delete',
-            title: STRINGS.LABELS.CHANNEL_MESSAGE_DELETE,
-            onPress: () => confirmDelete(msg),
-          });
-        }
-
-        if (channel.isGroupChannel() && sbOptions.uikit.groupChannel.channel.replyType === 'quote_reply') {
-          const disabled = Boolean(msg.parentMessageId);
+    }
+    if (!channel.isEphemeral) {
+      if (isMyMessage(msg, currentUserId) && msg.sendingStatus === 'succeeded') {
+        if (msg.isUserMessage()) {
           sheetItems.push({
-            // TODO: Implement disabled to bottom sheet
-            // disabled,
-            icon: 'reply',
-            // TODO: Add reply label
-            title: disabled ? 'Reply(disabled)' : 'Reply', //'STRINGS.LABELS.CHANNEL_MESSAGE_REPLY',
-            onPress: () => onReplyMessage?.(msg),
+            icon: 'edit',
+            title: STRINGS.LABELS.CHANNEL_MESSAGE_EDIT,
+            onPress: () => onEditMessage(msg),
           });
         }
+        sheetItems.push({
+          disabled: msg.threadInfo && msg.threadInfo.replyCount > 0,
+          icon: 'delete',
+          title: STRINGS.LABELS.CHANNEL_MESSAGE_DELETE,
+          onPress: () => confirmDelete(msg),
+        });
       }
+      if (channel.isGroupChannel() && sbOptions.uikit.groupChannel.channel.replyType === 'quote_reply') {
+        sheetItems.push({
+          disabled: Boolean(msg.parentMessageId),
+          icon: 'reply',
+          title: STRINGS.LABELS.CHANNEL_MESSAGE_REPLY(msg.sender),
```

Comment:
relates to https://github.com/sendbird/sendbird-uikit-react-native/pull/110/files#r1263426512
```suggestion
          title: STRINGS.LABELS.CHANNEL_MESSAGE_REPLY,
```